3.evaluate_model/scripts/nbconverted/apply_tsne.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Apply tsne to each feature dataset
+# 
+# Input: Data representations
+# Output: tsne embeddings for plotting
+
+# In[1]:
+
+
+import pathlib
+import numpy as np
+import pandas as pd
+from sklearn.manifold import TSNE
+
+import sys
+
+sys.path.append("../utils")
+from split_utils import get_features_data
+
+
+# In[2]:
+
+
+np.random.seed(1234)
```

Comment:
Why not random seed `0`? I have found that through previous PR reviews that Way Lab convention is to use `0` as random seed specificaly.

Reply:
we do not have a convention on random seeds - for whatever reason, i always use 1234